src/test/java/redis/clients/jedis/HostAndPorts.java

```diff
@@ -34,6 +35,10 @@ public final class HostAndPorts {
     clusterHostAndPortList.add(new HostAndPort("localhost", 7382));
     clusterHostAndPortList.add(new HostAndPort("localhost", 7383));
     clusterHostAndPortList.add(new HostAndPort("localhost", 7384));
+
+    stableClusterHostAndPortList.add(new HostAndPort("localhost", 10379));
+    stableClusterHostAndPortList.add(new HostAndPort("localhost", 10380));
+    stableClusterHostAndPortList.add(new HostAndPort("localhost", 10381));
```

Comment:
I'd prefer to keep these in 7000 series; i.e. `73XY` or `7X79-81`.

Reply:
Moved to 7479-7481.